### ISSUE TITLE
Hello, how did u get the rel_dist_mapped from the detector of neural-motifs?

### ISSUE THREAD

**Issue by taksau** [shikorab/SceneGraph#6]
Hi, Thanks for your sharing. I have one confusion here. How could you get the relation probability from the baseline (neural-motif)? It seems that we only can get the confidence/feature of the object without the LSTM and context.

**Comment by roeiherz** [comment]
Hey, thanks for your comment. So, I don't know if something has changed with their code lately, but 8 months ago, there was definitely a way to change the params, so that you can get out the relation logits/feature from there without LSTM and context. In their own paper, they also reports on the baseline (called MOTIFNET-NOCONTEXT), which is similar to our baselines results report. You should check their repo for running this baseline (MOTIFNET-NOCONTEXT) for a scene graph classification task.

**Comment by taksau** [comment]
Thanks for your explanation. Now, the detail of your paper is much more clearly for me.
